dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -357,9 +357,16 @@ public File downloadTargetEntry(String toolpath, ToolDescriptor.TypeEnum type, b
         // if no master is present (for example, for hosted workflows), fail over to the latest descriptor
         if (!first.isPresent()) {
             first = workflow.getWorkflowVersions().stream().max(Comparator.comparing(WorkflowVersion::getLastModified));
+            System.out.println("Could not locate workflow with version '" + tag + "'. Using last modified version '"
+                    + first.get().getName() + "' instead.");
```

Comment:
Is `first` guaranteed to be present? If not, you will get an error on `first.get`.

Reply:
OK added a check for whether 'first' is Present